### PR TITLE
Added DFWStartupCommunity to list

### PR DESCRIPTION
**Pull request by kstrib**
Closes issue #24 .

### CODE REVIEW

**Comment by kstrib** [comment]
This was my first ever pull request.. I'm not really sure what to do from here. I uploaded it to be checked and it passed, but it's saying that a review is required (but I don't know what that means or where to make that happen) and it says "merging is blocked" until one approved review. What do I need to do?

**Comment by itzsaga** [comment]
Hey @kstrib, so after opening your Pull Request you'll want to wait for a response from someone on the project. 99% that'll be me here. I'll work through all of these this week. If everything looks good it'll be merged in and closed.

File: src/list.html [hunk 1]
```diff
@@ -100,6 +100,8 @@ <h2 class="subtitle">
               <li><a href="http://slack.memphistechnology.org/" target="_blank">memtech</a> - Memphis technical community chat.</li>
               <li><a href="https://nashdev.com" target="_blank">NashDev</a> - A network for Nashville developers.</li>
             </ul>
+            <ul><h3>Texas</h3></ul>
```
Comment (itzsaga) [inline src/list.html]: This should be an `li` instead of a `ul`.

File: src/list.html [hunk 2]
```diff
@@ -100,6 +100,8 @@ <h2 class="subtitle">
...
+              <li><a href="http://slack.dfwstartupcommunity.com/" target="_blank">DFWStartupCommunity</a></li>
```
Comment (itzsaga) [inline src/list.html]: This needs to be wrapped by `<ul>` and `</ul>` tags.

Comment (kstrib) [reply src/list.html]: Thanks. I resolved these issues on my editor... but now it seems like I can't remember how to get it back to here.. (sigh). I'm working in ATOM which has everything GitHub integrated into it. I just forget the steps to take.

Comment (itzsaga) [reply src/list.html]: You'll need to `git add` them, then `git commit` them, and finally `git push` them. They will automatically show up on the PR.

Just leave a comment when the changes are made and I'll review. 😃

**Comment by itzsaga** [comment]
Okay, I haven't used Atom since the GitHub integration. However, do you have a terminal package installed? If not, search for and install the platformio-ide-terminal package. Then we can go from there. Also, you're part of the Slack channel right? It might be easier to go back and forth there for this.

**Pull request merged**